treeTable.js

```diff
@@ -218,10 +213,10 @@
 
     TreeTable.prototype.collapseAllRows = function () {
         const dt = this.$el.DataTable();
-        dt.rows().eq(0).filter((rowIdx) => {
-            const row = dt.row(rowIdx).data();
-            if (hasChild(this, row.tt_key)) {
-                this.collapsed.add(row.tt_key);
+
+        dt.rows((idx, data) => {
```

Comment:
Also discovered that using a function filter to query rows from datatables (https://datatables.net/reference/type/row-selector#Function) was quicker than getting all rows and using `Array.prototype.filter`